src/patch-events.js

```diff
@@ -266,11 +281,7 @@ export function addEventListener(type, fn, optionsOrCapture) {
   if (fn.__eventWrappers) {
     // Stop if the wrapper function has already been created.
     for (let i = 0; i < fn.__eventWrappers.length; i++) {
-      if (fn.__eventWrappers[i].node === this &&
```

Comment:
This is the PR,with test case https://github.com/webcomponents/shadydom/pull/35#issuecomment-258045254